ckanext/sse/password_policy.py

```
@@ -0,0 +1,1375 @@
+    """The union of every configured/bundled blocklist file."""
+    words = set()
+    for path in _blocklist_paths():
+        words |= _read_blocklist_file(path)
+    return words
+
+
+def hibp_check():
+    """Whether to check candidates against Have I Been Pwned (opt-in).
+
+    Off by default: it is an outbound call per password change and needs
+    network egress. The bundled/​configured word lists cover the offline case;
```

Ruff (0.16.3)

[error] Invalid unescaped character zero-width-space, use "\u200B" instead

Replace with escape sequence

(PLE2515)